lib/messages/builder.js

```diff
@@ -68,6 +68,9 @@ function builder(options) {
       ix: 'TXLockRequest'
     },
     unsupportedCommands: [
+      'qsendrecsigs',
+      'senddsq',
+      'sendcmpct',
```

Comment:
I'm not sure if I missed things somewhere else, but there are still quite a few p2p messages not included here (e.g. `blocktxn`, `cmpctblock`, `mnauth`, `islock`, `clsig`, most of the messages in the [quorum messages](https://dash-docs.github.io/en/developer-reference#quorum-messages) section). Is the goal to have a comprehensive list?

Reply:
This PR was just focussing on getting dashcore-p2p starting up without crashing. I just added these three messages because them not being there was causing it to error before a stable state was found. I would definitely like to have a comprehensive list eventually. But would like to add more messages in a seperate PR and just do the minimum here to have it running again.

Reply:
Sounds good. Are `islock`, `clsig`, and (potentially) `qsigrec` going to become an issue once Sporks 19/20 are activated? I guess these wouldn't be startup issues per se, but those messages will be getting relayed through the network after the spork activation.

Reply:
I've been running dashcore-p2p on testnet, which has Sporks 19/20 activated, now for a few hours and there were no errors/crashes. So I guess we are good. After the sporks have been activated on mainnet I will test again if there are any issues. And then make a new PR for all the remaining messages you've mentioned, `blocktxn`, `cmpctblock`, `mnauth`, `islock`, `clsig` plus anthing else we find in addition to this.

Reply:
Great!